doc/user/content/guides/node-js.md

```diff
@@ -1,23 +1,21 @@
 ---
 title: "Node.js and Materialize"
-description: "This guide explains how to use a Node.js application to connect, insert data, manage, and read data from Materialize."
+description: "Reference and examples on using a Node.js application to connect, insert data, manage, and read data from Materialize."
 weight: 10
```

Comment:
The problem is that it's "Reference on" and "Examples of". I would just make the entire phrase "Use". Putting it in the imperative also signals to users that it's about how to do particular things.